src/api/artifact/abstractor/resolver/chart/chart.go

```diff
@@ -9,3 +13,68 @@
 // limitations under the License.
 
 package chart
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/goharbor/harbor/src/api/artifact/abstractor/blob"
+	resolv "github.com/goharbor/harbor/src/api/artifact/abstractor/resolver"
+	"github.com/goharbor/harbor/src/common/utils/log"
+	"github.com/goharbor/harbor/src/pkg/artifact"
+	"github.com/goharbor/harbor/src/pkg/repository"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+const (
+	// ArtifactTypeChart defines the artifact type for helm chart
+	ArtifactTypeChart = "CHART"
+)
+
+func init() {
+	resolver := &resolver{
+		repoMgr:     repository.Mgr,
+		blobFetcher: blob.Fcher,
+	}
+	if err := resolv.Register(resolver, "application/vnd.cncf.helm.config.v1+json"); err != nil {
```

Comment:
For such value, we should have a set of const values for easy reference. Even including the value like `ArtifactTypeChart = "CHART"`

Reply:
The media types of images are imported from distribution/oci packages, we didn't define them in Harbor.

For the `ArtifactTypeChart`, it's defined by the resolvers themselves, so define it in the resolver's package is reasonable